drivers/adxl345/Kconfig

```
@@ -5,8 +5,14 @@
 # directory for more details.
 #
 
-config MODULE_ADXL345
+menuconfig MODULE_ADXL345
```

It has no configurations inside
```suggestion
config MODULE_ADXL345
```

---

I set everything to menuconfig as adding things later would be a bit cleaner and I don't know if any functional difference.

If you feel strongly about it I will revert everything to `config` only.